internal/server/interceptors/cors.go

```diff
@@ -0,0 +1,36 @@
+package interceptors
+
+import (
+	"net/http"
+	"strings"
+)
+
+func AllowCORS(h http.Handler, origin string) http.Handler {
```

Comment:
Maybe we can use mux middleware for it? https://github.com/gorilla/handlers/blob/546854cf1d61a016260b224e2526a29ff5b5c5ae/cors.go